starters/adaptors/azure-swa/swa-cli.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://aka.ms/azure/static-web-apps-cli/schema",
+  "configurations": {
+    "qwik-app": {
+      "appLocation": ".",
+      "apiLocation": ".",
+      "outputLocation": "./dist",
+      "appBuildCommand": "npm run build",
+      "apiBuildCommand": "npm run build.server"
+    }
+  }
+}
```

Comment:
In addition to this file, it may be helpful to scaffold out an example `azure-static-web-apps.yml` file, like this one:
https://github.com/derkoe/qwik-swa-demo/blob/main/.github/workflows/azure-static-web-apps-salmon-dune-02a16c603.yml
Without it, I wouldn't have found the right combination of the build configuration values.

Reply:
I'm not sure about that because then the GitHub action will fail as soon as a developer pushes the repo. The setup via the Azure portal is pretty easy and will create the necessary file for you. (Azure exports may create their own pipelines).

But if you want to have I can add it - should I?

Reply:
Good point, perhaps not. Having the Github action fail immediately isn't desirable behavior, might be more confusing.